docs/develop/automatically-open-a-task-pane-with-a-document.md

```diff
@@ -105,11 +105,12 @@ The `webextension` part includes a property bag and a property named **Office.Au
 
 The `webextension` part also includes a reference to the store or catalog with attributes for `id`, `storeType`, `store`, and `version`. Of the `storeType` values, only four are relevant to the autoopen feature. The values for the other three attributes depend on the value for `storeType`, as shown in the following table.
 
-| **`storeType` value** | **`id` value**    |**`store` value** | **`version` value**|
+|`storeType` value|`id` value|`store` value|`version` value|
 |:---------------|:---------------|:---------------|:---------------|
-|OMEX (AppSource)|The AppSource asset ID of the add-in (see Note)|The locale of AppSource; for example, "en-us".|The version in the AppSource catalog (see Note)|
+|OMEX (AppSource)|The AppSource asset ID of the add-in (see Note).|The locale of AppSource; for example, "en-us".|The version in the AppSource catalog (see Note).|
+|WOPICatalog (third-party [WOPI](/microsoft-365/cloud-storage-partner-program/online/) hosts)| The AppSource asset ID of the add-in (see Note). | "wopicatalog". Use this value for add-ins that are published in App Source and are installed in third-party WOPI hosts. For more information, see [Enable Office Web Add-Ins provided by Host](/microsoft-365/cloud-storage-partner-program/online/overview). | The version in the add-in manifest.|
```

Comment:
Since the second link goes to a different article, should the title noted here match that?
```suggestion
|WOPICatalog (third-party [WOPI](/microsoft-365/cloud-storage-partner-program/online/) hosts)| The AppSource asset ID of the add-in (see Note). | "wopicatalog". Use this value for add-ins that are published in App Source and are installed in third-party WOPI hosts. For more information, see [Integrating with Office Online](/microsoft-365/cloud-storage-partner-program/online/overview). | The version in the add-in manifest.|
```

Reply:
Yeah, thanks for catching that.